libexec/bats-core/bats-exec-test

```diff
@@ -43,6 +43,12 @@ elif [[ ! -f "$BATS_TEST_FILENAME" ]]; then
   exit 1
 fi
 
+if ! BATS_TEST_TMPDIR=$(mktemp -d "${BATS_FILE_TMPDIR}/bats-test-tmpdir-XXXXXX");then
```

Comment:
Likewise to the comment on BATS_FILE_TMPDIR it is more readable to have `${BATS_FILE_TMPDIR}/test-tmpdir-${BATS_TESTNAME}-XXXX`.